### PR TITLE
Add semi-automated misc classification for providers release

### PR DESCRIPTION
**Pull request by eladkal**

### CODE REVIEW

File: dev/README_RELEASE_PROVIDER_PACKAGES.md [hunk 1]
```diff
@@ -79,13 +79,24 @@ You can read more about the command line tools used to generate the packages in
...
+2. Check if Breeze unit tests in `dev/breeze/tests/test_packages.py` need adjustments. This is done by simply
+searching and replacing old version occurrences with newer one. For example 2.5.0 to 2.6.0
+
+3. Update minimum airflow version for all packages, you should modify `MIN_AIRFLOW_VERSION`
+in `src/airflow_breeze/utils/packages.py` and run the `prepare-provider-documentation`
 command with the `--only-min-version-update` flag. This will only update the min version in
 the `__init__.py` files and package documentation without bumping the provider versions.
 
 ```shell script
+branch="Bump minimum Airflow version in providers to Airflow 2.6.0"
```
Comment (hussein-awala) [inline dev/README_RELEASE_PROVIDER_PACKAGES.md:94]: This is not a valid branch name.

Comment (eladkal) [reply dev/README_RELEASE_PROVIDER_PACKAGES.md:94]: Copy paste :)
Will fix it

**Comment by potiuk** [comment]
Nice one :)

**Review by potiuk — APPROVED** [review]

File: dev/README_RELEASE_PROVIDER_PACKAGES.md (hunk 1 above)
Comment (eladkal) [reply dev/README_RELEASE_PROVIDER_PACKAGES.md:94]: https://github.com/apache/airflow/pull/36020

**Pull request merged**